gatsby/src/components/looking-for-something/looking-for-something.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import classes from './looking-for-something.module.scss';
+import { ArrowForwardIos, Phone } from '@material-ui/icons';
+
+const defaultValues = {
+  searchingHeader: (
+    <>
+      <strong>Stále</strong> hledáte?
+    </>
+  ),
+  searchingDescription:
+    'Nenašli jste odpověď v žádné kategorii, tématu ani častých otázkách?',
+  callHeader: (
+    <>
+      Volejte{' '}
+      <strong>
+        <a className={classes.phoneNumber} href="tel:1221">
+          1221
+        </a>
+      </strong>
+    </>
+  ),
+  callDescription: 'Pro urychlení si prosím připravte svůj konkrétní dotaz.',
+};
+
+type Props = typeof defaultValues;
```

Comment:
please type the component manually,
soon we will remove the `defaultValues` so the types will stop working :(
could you please define props as:
```
type Props = {
  title: string, // leave here just "Stale hledate?" I will cover it later
  description: string,
  callTitle: string,
  callNumber: string,
  callDescription: string
}
```

Reply:
Sorry went to the bed already :-( Hopefully you will cover it later.